src/main/java/com/feelmycode/parabole/domain/User.java

```diff
@@ -53,16 +53,19 @@ public class User extends BaseEntity {
     @OneToMany(mappedBy = "user")
     private List<UserCoupon> userCoupons = new ArrayList<>();
 
-    /** 연관관계 메서드 */
-    public void setUserCoupon(UserCoupon userCoupon){
-        userCoupon.setUser(this);               // owner
-        this.getUserCoupons().add(userCoupon);
-    }
-
+    /** User 에게 판매자 권한을 부여하는 것을 오직 Admin 만 가능하기에 함수 사용할 일 X */
     public void setSeller(Seller seller) {
+        seller.setUser(this);
         this.seller = seller;
     }
 
+    public String checkRole() {
+        if (seller != null) {
+            return "SELLER";
+        }
+        return "USER";
+    }
+
```

Comment:
이 부분을 SELLER아니면 USER값만 가져오니까 boolean으로 처리해두는게 좋을거 같아요
```suggestion
    public boolean isUser() {
        if (seller != null) {
            return true;
        }
        return false;
    }

```

Reply:
호출되는 부분은 없지만 편의를 위해 User Entity 안에도 추가해준 메서드 입니다 ~ 반영했습니다